### DISCUSSION TITLE
AddCommand

### DISCUSSION THREAD

**Discussion by alertya** [TannerGabriel/discord-bot#182] [Ideas]
Hi,

it would be amazing if we could have more commands than "play", "shuffle", "skip",etc

I was trying to implement one called Shortplay but unfortunely the Bot does not recognize it as an interaction,
do u know what procedure should I do for that ?

Best regards,

**Comment by TannerGabriel** [comment]
Did you follow the command structure ([example](https://github.com/TannerGabriel/discord-bot/blob/master/commands/play.js#L4-L15)) and use the `!deploy` command after restarting the bot?

**Reply by alertya** [reply]
Ahh sorry for the inconvenience, yes I did the structure command but totally forgot to use the deploy command, it works totally fine right now. Thanks for your work and contribution.
